### PR TITLE
Add image/scripts to run migrations in docker

### PR DESCRIPTION
**Pull request by langchain-infra**

### CODE REVIEW

File: charts/langsmith/scripts/backfill_clickhouse.sh [hunk 1]
```diff
@@ -114,7 +114,7 @@ SELECT
     0, -- is_deleted
     toString(c26), -- inputs_s3_urls
     toString(c27) -- outputs_s3_urls
-FROM input('c1 String, c2 String, c3 String, c4 DateTime64(6), c5 DateTime64(6), c6 String, c7 String, c8 bool, c9 String, c10 String, c11 String, c12 String, c13 String, c14 String, c15 String, c16 String, c17 String, c18 String, c19 String, c20 String, c21 String, c22 UInt64, c23 UInt64, c24 UInt64, c25 DateTime64(6), c26 String, c27 String')
```
Comment (langchain-infra) [inline charts/langsmith/scripts/backfill_clickhouse.sh:117]: fix first token time showing up as a negative number when null

File: charts/langsmith/docs/UPGRADE-0.2.x.md [hunk 2]
```diff
@@ -71,12 +71,81 @@ langsmith-queue-d58cb64f7-87d68          1/1     Running     0          15h
 
     ![.langsmith_ui.png](../langsmith_ui.png)
 
-Note that old runs will not be visibile in the UI unless you migrate them. See [Migrating old runs](#migrating-old-runs) for more information.
+Note that old runs will not be visibile in the UI unless you migrate them. See [Migrating old runs](#migrating-old-runs-) for more information.
```
Comment (nfcampos) [inline charts/langsmith/docs/UPGRADE-0.2.x.md:74]: This - on purpose?

**Review by nfcampos — APPROVED** [review]

File: charts/langsmith/docs/UPGRADE-0.2.x.md (hunk 2 above)
Comment (langchain-infra) [reply charts/langsmith/docs/UPGRADE-0.2.x.md:74]: oops, good catch

Comment (langchain-infra) [reply charts/langsmith/docs/UPGRADE-0.2.x.md]: fixed

**Pull request merged**